docs/download/repositories.md

```diff
@@ -0,0 +1,124 @@
+
+
+#### Download Client Operation
+
+To operate the EGA download client, follow the instructions [here](https://ega-archive.org/download/using-ega-download-client#DownloadClient).
+
+## [GDC](https://gdc.nci.nih.gov)
+
+The Genomic Data Commons is a US government ([NIH](https://www.nih.gov/) / [NCI](https://www.cancer.gov/)) run data repository for cancer genomic information. Notably, the it carries data from The Cancer Genome Atlas (TCGA) and the Therapeutically Applicable Research to Generate Effective Treatments (TARGET). Currently the GDC is the largest single repository of ICGC data. It focuses on studies in the United States.
+
+| Property             | Value                                                                                                                                  |
+| :------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Name                 | Genomic Data Commons                                                                                                                   |
+| Governing Body       | National Cancer Institute                                                                                                              |
+| Contact              | <mailto:support@nci-gdc.datacommons.io>                                                                                                |
+| Official Data Portal | <https://portal.gdc.cancer.gov/>                                                                                                      |
+| ICGC Portal Page     | [Portal](https://dcc.icgc.org/repositories?filters=%7B%22file%22:%7B%22repoName%22:%7B%22is%22:%5B%22GDC%20-%20Chicago%22%5D%7D%7D%7D) |
+| Download Client      | Download the client [here](https://gdc.cancer.gov/access-data/gdc-data-transfer-tool)                                                                 |
+| Repo Code            | `gdc`                                                                                                                                  |
+
+#### Obtaining Data Access
+
+To obtain access you must have an eRA commons account, and you must have dbGap access to the data on the GDC you are interested in. Talk to your team leader if you do not have this access. Once this account is set up, you can log in to the GDC **using the your dbGaP credentials**.
+
+<https://gdc.cancer.gov/access-data/obtaining-access-controlled-data>
+
+#### Download Client Operation
+
+Once you or your project leader have attained access to the reseach project,you will need to download access tokens from the gdc data portal. A comprehensive guide on how to use the GDC client is available [here.](https://gdc-docs.nci.nih.gov/Data_Transfer_Tool/Users_Guide/Preparing_for_Data_Download_and_Upload/)
```

Comment:
The link https://gdc-docs.nci.nih.gov/Data_Transfer_Tool/Users_Guide/Preparing_for_Data_Download_and_Upload/ does not work. The correct link should be https://gdc.cancer.gov/access-data/gdc-data-transfer-tool

Reply:
fixed